examples/spark/images/base/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM java:latest
+FROM java:openjdk-8-jdk
```

Comment:
Yeah, it seems to be function on JDK8, but I could kick it back if you want. Sorry, I did notice that pre-req as well and presumed it meant 1.7+. Other parts of that page are slightly in need of editing, too.